src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/IEmployerDashboardService.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Services
 {
-    public interface IDashboardService
+    public interface IEmployerDashboardService
```

Comment:
Could this be left as `IDashboardService` as the other dashboards could also implement this interface and the method names are general.

Reply:
👍